src/Components/Programs/styles.js

```diff
@@ -0,0 +1,85 @@
+import { View, Text ,ScrollView} from 'react-native';
+import { styled } from 'react-native-reflect';
+export const Event = styled(Text,{
+    fontSize: 24,
+    borderWidth: 0.5,
+    paddingVertical:4,
+    paddingHorizontal: 8,
+    marginBottom: 4,
+    borderRadius: '50px',
+    textAlign: 'center',
+    zIndex:'1px',
+    position:'relative',
+    backgroundColor:'#fff',
+    width:'200px'
+});
+export const Line = styled(View,{
+    width:'200vw',
+    borderBottomWidth:10,
+    marginBottom:4,
+    position:'relative',
+    top: '18px',
+    left: '20%',
+});
+export const Date = styled(View,{
+    backgroundColor:'#ffdd30',
+    color:'#fff',
+    fontSize:20,
+    width:45,
+    height:45,
+    borderRadius: '50px',
```

Comment:
we don't use px because this will break moving to modules to native app.

Reply:
Without the 'px' border-radius doesn't seem to work. The same problem occurs for z-index also.

Reply:
@annabauza